mbuild/formats/lammpsdata.py

```diff
@@ -54,7 +54,14 @@ def write_lammpsdata(structure, filename, atom_style='full'):
     # Internally use nm
     box = Box(lengths=np.array([0.1 * val for val in structure.box[0:3]]),
               angles=structure.box[3:6])
-
+    """
+    Note:
+    -----
+    unique_types : a sorted list of unique atom types for all atoms in the structure.
+    unique_bond_types: an enumarated OrderedDict of unique bond types for all bonds in the structure. Unique bond type is defined by bond parameters, in order, `k`, `req`, and a sorted string of atom types of component atoms.
```

Comment:
These lines go on for quite a bit. Can you turn these into multi line comments for each variable?